scripts/pugme.js

```diff
@@ -0,0 +1,41 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, msg => msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => msg.send(JSON.parse(body).pug)));
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
+    let count;
```

Comment:
This count can also be converted into `const`. You can just do something like 

```javascript
const count = (parseInt(msg.match[1], 10) > 10) ? 10 : msg.match[1] || 5;
```

Little contrived so I'll leave this one to you but it lets us get around getting rid of `let`s from the entire code which I far prefer.

Reply:
Oh shoot, I forgot to put a `parseInt` in the other case myself. Oops.